medusa/providers/torrent/html/anidex.py

```diff
@@ -36,7 +36,7 @@ def __init__(self):
 
         # Miscellaneous Options
         self.supports_absolute_numbering = True
-        self.anime_only = True
+        self.anime_only = False
```

Comment:
Same here. You can remove to make it opt-in like the other providers.

Reply:
Already done ;)